otter/worker/launch_server_v1.py

```diff
@@ -308,6 +308,13 @@ def _check_server_created(f):
         d.addBoth(_check_results, f)
         return d
 
+    def _create_with_delay():
+        d = _treq.post(path, headers=headers(auth_token),
+                       data=json.dumps({'server': server_config}), log=log)
+        # Add 1 second delay to space 1 second between server creations
+        d.addCallback(delay, clock, 1)
```

Comment:
I don't think that this should block this PR right now, FWIW :-)

There are two ways to do this:

- Supervisor, as you were suggesting. That would space out *jobs*, not server creations; not quite what Compute is asking for, and, in the problematic cases, not something that actually performs their experiment.
- Separate the concern of "one at a time" from "at most one per second". I.e., don't change the thing you're changing here at all; that fn creates a server, and it still creates a server (no added delay). Instead, Make the delay be about adding things to the `DeferredSemaphore` in the first place, by e.g. putting create server requests in a queue, and consuming them every *timeout* seconds.